### ISSUE TITLE
[BUG] METHOD_CALL doesn't work with methods in embedded classes

### ISSUE THREAD

**Issue by jmle** [konveyor/analyzer-lsp#808]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Konveyor version

0.0.6

### Priority

Major

### Current Behavior

Methods like `ExpressionUrlAuthorizationConfigurer.ExpressionInterceptUrlRegistry.access(String)` are not detected.

### Expected Behavior

Embedded classes and methods should be able to be detected

### How Reproducible

Always (Default)

### Environment

```markdown
- **Analyzer Image SHA** 
- **JDTLS  base image SHA**
- **Provider Settings JSON**
```
